ui/shared/components/panel/variants/Pathogenicity.jsx

```diff
@@ -65,6 +65,16 @@ PathogenicityLink.propTypes = {
   popup: PropTypes.string,
 }
 
+const ClinvarSubmissions = React.memo(({ submissions }) => submissions != null &&
```

Comment:
rather than iterating the list and returning null for all but one value, it might be neater to instead just get the broad submission and show the label based on that.
So in the main component something like 
```
const broadSubmission = submissions?.find(({ submitter }) => submitter === 'Broad...')
...
{broadSubmission && <BroadSubmissionLabel />}
```